javascript/ql/src/semmle/javascript/Expr.qll

```diff
@@ -409,6 +409,9 @@ class BigIntLiteral extends @bigintliteral, Literal {
  */
 class StringLiteral extends @stringliteral, Literal {
   override string getStringValue() { result = getValue() }
+
+  /** Gets the value of this string literal parsed as a regular expression. */
```

Comment:
If I understand the extractor changes correctly, not all string literals are parsed as regular expressions. Would it be worth explaining this briefly in the doc comment?